lib/stat_tracker.rb

```diff
@@ -34,4 +37,20 @@ def total_goals_per_game
   def highest_total_score
     total_goals_per_game.max_by {|game_id, total_goals| total_goals}[1]
   end
-end
\ No newline at end of file
+
+  def count_of_games_by_season
+    season_games = Hash.new(0)
+    @games.each do |game|
+      season_games[game.season] += 1
+    end
+    season_games
+  end
+
+  def average_goals_per_game
+    result = total_goals_per_game.values.sum do |value|
+      value
+    end
```

Comment:
@sarynm12 I think right here you could just do `result = total_goals_per_game.values.sum` without `do |value|` or line 51-52.

Reply:
You're totally right!Thank you. I just updated that!